server/views/pages/activities/movement-list/location-events.njk

```diff
@@ -26,30 +26,29 @@
                             {{ event.summary | trim }}
                         </a>
                     </div>
-                    {% if event.suspended %}
-                        <div class="govuk-!-margin-top-1">{{ govukTag({ text: 'Prisoner suspended', classes: 'govuk-tag--red-border govuk-tag--small' }) }}</div>
-                    {% endif %}
                 {% elseif event.eventType == EventType.APPOINTMENT%}
                     <div>
                         <a href="/appointments/{{ event.appointmentId }}"
                            class="govuk-link govuk-link--no-visited-state" target="_blank">
                             {{ event.summary | trim }}
                         </a>
                     </div>
-                    <div>
-                        {% if event.cancelled %}
-                        <div>{{ govukTag({ text: 'Cancelled Appointment', classes: 'govuk-tag--small govuk-tag--red' }) }}</div>
-                        {% else %}
-                            <div>{{ govukTag({ text: 'Appointment', classes: 'govuk-tag--small' }) }}</div>
-                        {% endif %}
-                    </div>
+                    <div>{{ govukTag({ text: 'Appointment', classes: 'govuk-tag--small' }) }}</div>
                     {% if event.comments %}
                         <div>{{ govukTag({ text: 'Extra information', classes: 'govuk-tag--small govuk-tag--yellow govuk-!-display-none-print' }) }}</div>
                     {% endif %}
                 {% else %}
                     <div>{{ event.summary | trim }}</div>
                 {% endif %}
 
+                {% if event.cancelled %}
+                    <div>{{ govukTag({ text: 'Cancelled', classes: 'govuk-tag--small govuk-tag--red' }) }}</div>
```

Comment:
I noticed you have a margin-top-1 class on the suspended tag below – is it needed here too?